api/employees.py

```diff
@@ -1,21 +1,26 @@
 from hashlib import md5
 from typing import Annotated, List
 
+from fastapi.security import OAuth2PasswordBearer
+
 from globals import Globals
 from library.data.local.neo4j import Neo4j
+from library.managers.auth_manager import AuthManager
 from library.models.api_models import ApiResponse
 from library.managers.api_support import APISupport
 
-from library.models.employee import Employee
+from library.models.employee import Employee, User
 from library.managers.employee_manager import EmployeeManager
-from fastapi import APIRouter, Body, UploadFile
+from fastapi import APIRouter, Body, Depends, UploadFile
 
 from library.workday import Workday
 
 route = APIRouter(tags=["Employees"])
+oauth2_scheme = OAuth2PasswordBearer(tokenUrl="auth/login/openapi")
 
 @route.put('/employees')
-async def add_employees_from_workday_export(workday_csv_export: UploadFile) -> ApiResponse[str]:  
+async def add_employees_from_workday_export(me: Annotated[User, Depends(AuthManager.get_user_dependency(oauth2_scheme))],
+                                            workday_csv_export: UploadFile) -> ApiResponse[str]:  
```

Comment:
It's cool to see the "Depends" mechanism which helps us ensure that only authenticated users can access these endpoints.